### ISSUE TITLE
Make it so the default is to not show indirect translations in searches.

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1909]
## The Problem

When visitors and new members who don't understand what indirect translations are see them, they get the impression that there are a lot of errors on the website.

## Some Possibilities.

1. Make it so searches by non-logged-in members never show indirect translations.

2. Make it so that the default even for members is this, but make it a choice to see indirect translations in their settings.

3. Make it so the default is to not show indirect translations, but include a checkbox that would enable search results to also show indirect translations, with a cookie remembering the choice.  This is what I'd prefer, since there are times when I'd like to see the indirect translations, and other times when I don't.

<img width="1188" alt="also show indirect translations" src="https://user-images.githubusercontent.com/7934273/60145274-8036ae80-9800-11e9-9119-dfc06842910c.png">

"Also show indirect translations"
With a little thought, we could possibly come up with a more concise wording for this.

## How This Came to My Attention.

Someone testing this page http://www.manythings.org/tatoeba/eng/ (How to Use the Tatoeba Corpus to Study English) clicked on the "study - random" link and got Japanese equivalents that obviously were "wrong.) (That is, they were wrong because the tester didn't realize that they were translations of translations.)

Here's the actual link.  However, since it's a random selection, you may not see the same problem.

https://tatoeba.org/sentences/search?trans_filter=limit&trans_link=direct&from=eng&to=jpn&list=907&has_audio=yes&trans_to=jpn&sort=random

**Comment by trang** [comment]
@ckjpn Was your student logged in and using the old sentence design?

I think the solution should rather be to make it clearer that indirect translations won't always exactly match the main sentence. Or to make indirect translations more distinguishable. This depends if your student actually didn't notice it was indirect translations, or if they did see that it was indirect translations but it was not intuitive to them that indirect translations won't always match.

The feature of indirect translation is (to me at least) a very important feature of Tatoeba. It allows to connect languages that would otherwise not be connected.

For instance, I searched [French sentences translated into Vietnamese](https://tatoeba.org/eng/sentences/search?from=fra&to=vie&query=), and there are 5371 occurrences. If I would limit that search to [direct translations only](https://tatoeba.org/eng/sentences/search?query=&from=fra&to=vie&orphans=no&unapproved=no&user=&tags=&list=&has_audio=&trans_filter=limit&trans_to=vie&trans_link=direct&trans_user=&trans_orphan=&trans_unapproved=&trans_has_audio=&sort=relevance), there are 1279 occurrences, which is around 25% of the previous search. Unless the majority of the indirect translations are dreadfully wrong, it seems like disproportionate measure to exclude them by default.

**Comment by trang** [comment]
Closing since we lack information.

Maybe we find out more about the issue by doing more UX tests, but in the meantime we will definitely solve this by removing indirect translations from the default search.
